presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -1358,16 +1358,16 @@ public HiveClientConfig setS3SelectPushdownMaxConnections(int s3SelectPushdownMa
         return this;
     }
 
-    public boolean isStreamingAggregationEnabled()
+    public boolean isOrderBasedExecutionEnabled()
     {
-        return streamingAggregationEnabled;
+        return orderBasedExecutionEnabled;
     }
 
-    @Config("hive.streaming-aggregation-enabled")
-    @ConfigDescription("Enable streaming aggregation execution")
-    public HiveClientConfig setStreamingAggregationEnabled(boolean streamingAggregationEnabled)
+    @Config("hive.order-based-execution-enabled")
+    @ConfigDescription("Enable order-based execution. When it's enabled, hive files become non-splittable")
```

Comment:
And the ordering property of the table is exposed to the optimizer to plan streaming aggregation and merge join when possible.